scss/_variables.scss

```diff
@@ -622,8 +622,8 @@ $border-radius-pill:          50rem !default;
 // scss-docs-end border-radius-variables
 
 // fusv-disable
-$outline-width:               var(--#{$prefix}border-width) !default; // Deprecated in v5.2.3
-$outline-offset:              $outline-width !default; // Deprecated in v5.2.3
+$outline-width:               var(--#{$prefix}border-width) !default; // Deprecated in v5.3.0-alpha1
+$outline-offset:              $outline-width !default; // Deprecated in v5.3.0-alpha1
```

Comment:
If I haven't made any mistakes, it is fixed in a 2-step approach (because this is the morning :D) via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1826/commits/c15214cfa6472b701ced9d20745aee24cd7b922a and https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1826/commits/0aac053815bab5e59a6f47f546854a2064eee789